lib/string-utils.js

```diff
@@ -0,0 +1,9 @@
+const getLink = str => {
+  const containsLink = str.match(/\((?<url>https?:.*)\)/);
```

Comment:
Argh. Node 8 doesn't support named capture groups ☹️ 

We should go back to numbered groups the i suppose.

Reply:
Yep 😢 it's not supported in node v8.

I have reverted the changes. Let me know if you prefer another approach.